### ISSUE TITLE
WebDriverException thrown from SendKeys() Selenium Grid

### ISSUE THREAD

**Issue by JoshuaBurns** [SeleniumHQ/selenium#2634]
## Meta - Node Configuration
OS:  Windows Server 2008 R2 6.1  |  v2.53.1  |  ra36b8b1

Selenium Version:  
Selenium standalone server 2.53.1
Selenium.WebDriver 2.53.1 
Selenium.Support 2.53.1
Selenium.WebDriver.IEDriver 2.53.1.1

Browser:  Internet Explorer

Browser Version:  11

I am attempting to get my tests running in parallel on the Selenium Grid. My Internet Explorer tests consistently fail on the SendKeys() call giving me the following exception. 

```
2016-08-19 16:44:33,256 DEBUG Test - OpenQA.Selenium.WebDriverException: The HTTP request to the remote WebDriver server for URL http://10.34.161.112:5555/wd/hub/session/0febecc5-3692-4f44-a263-70702f2d86b1/element/1/value timed out after 60 seconds. ---> System.Net.WebException: The operation has timed out
   at System.Net.HttpWebRequest.GetResponse()
   at OpenQA.Selenium.Remote.HttpCommandExecutor.CreateResponse(WebRequest request)
   --- End of inner exception stack trace ---
   at OpenQA.Selenium.Remote.HttpCommandExecutor.CreateResponse(WebRequest request)
   at OpenQA.Selenium.Remote.HttpCommandExecutor.Execute(Command commandToExecute)
   at OpenQA.Selenium.Remote.RemoteWebDriver.Execute(String driverCommandToExecute, Dictionary`2 parameters)
   at OpenQA.Selenium.Remote.RemoteWebElement.SendKeys(String text)
   at POMAuctivaTest.TestSuite.Test.setUp() in c:\git\POMAuctivaTest\POMAuctivaTest.TestSuite\Test.cs:line 35
```

I have tested locally and this is only an issue when ran on the grid. I was able to reproduce the issue on a non-internal site. Although the node is internal I hope you guys can reproduce the issue, here is the code. 

```
using System;
using NUnit.Framework;
using OpenQA.Selenium;
using OpenQA.Selenium.Support.UI;
using OpenQA.Selenium.Remote;
using OpenQA.Selenium.IE;
using POMAuctivaTest.AuctivaPageModels;

namespace POMAuctivaTest.TestSuite
{
    [TestFixture]
    public class Test
    {
        public IWebDriver driver { get; set; }
        [OneTimeSetUp]
        public void setUp()
        {
                DesiredCapabilities capabilities = DesiredCapabilities.InternetExplorer();
                capabilities.SetCapability(CapabilityType.BrowserName, "internet explorer");
                capabilities.SetCapability(CapabilityType.Platform, new Platform(PlatformType.Windows));
                capabilities.SetCapability(CapabilityType.Version, "11");
                driver = new RemoteWebDriver(new Uri("http://10.34.161.112:5555/wd/hub"), capabilities);
                //driver = new InternetExplorerDriver();
                driver.Manage().Timeouts().SetPageLoadTimeout(TimeSpan.FromSeconds(30));
                driver.Manage().Window.Maximize();
                driver.Navigate().GoToUrl("https://passport.yandex.com/passport?mode=auth&retpath=https%3A%2F%2Fmail.yandex.com%2Flite%2Finbox");
                By usernameSelector = By.CssSelector("#login");
                By passwordSelector = By.CssSelector("#passwd");
                By loginButtonSelector = By.CssSelector("body > div > div.layout-inner > div.domik-wrap > div > div.domik-content > form > div.domik-row > div > button");
                WebDriverWait wait = new WebDriverWait(driver, TimeSpan.FromSeconds(30));
                wait.Until(ExpectedConditions.ElementExists(loginButtonSelector));
                driver.FindElement(usernameSelector).SendKeys("pomauctivaie@yandex.com");
                driver.FindElement(passwordSelector).SendKeys("Bugzilla1!");
                driver.FindElement(loginButtonSelector).Click();
        }

        [Test]
        public void testIE()
        {            
                driver.Manage().Window.Maximize();
                System.Threading.Thread.Sleep(5000);          
        }

        [OneTimeTearDown]
        public void tearDown()
        {
            driver.Quit();
        }
    }
}
```

**Comment by JoshuaBurns** [comment]
Ok so this is not a new issue, after remote desktoping to my node I can see that I am having the slow typing issue on IE. 
I am running a 64-bit version of Internet Explorer and I have the changed over to 
`<package id="Selenium.WebDriver.IEDriver64" version="2.53.0.0" targetFramework="net45" />`
but am still seeing typing go really slow and causing WebDriverExceptions, is there something I am missing?

**Comment by JoshuaBurns** [comment]
This is not a bug sorry to pollute the boards, this can be closed.
